src/Core/Addon/Module/ModuleManager.php

```diff
@@ -388,11 +385,24 @@ public function enable($name)
      * Not written in camel case because the route and the displayed action in the template
      * are related to this function name.
      *
+     * @deprecated use disableMobile()
```

Comment:
Yeah, I understood that from the comment :)

However, since they aren't compliant with PSR-2, those methods names will have to be changed eventually (not necessarily right now)